packages/boot/src/booters/controller.booter.ts

```diff
@@ -0,0 +1,125 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/boot
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {CoreBindings, Application, Booter, BootOptions} from '@loopback/core';
+import {inject} from '@loopback/context';
+import {BootBindings} from '../keys';
+import {relative} from 'path';
+import {
+  discoverFiles,
+  discoverFilesWithGlob,
+  loadClassesFromFiles,
+} from './booter-utils';
+
+/**
+ * ControllerBooter is a class that implements the Booter inferface. The purpose
+ * of this booter is to allow for convention based booting of `Controller`
+ * artifacts for LoopBack 4 Applications.
+ *
+ * It supports the following boot phases: config, discover, boot
```

Comment:
Is the default order of the boot phases the same as the order of this list?  If so, change this to say
`The following boot phases are supported, in this order: config, discover, boot`

Reply:
The default order is `config, discover, boot` but this can be changed by the user by setting the `phases` array. I'm just listing the implemented `phases` so users know which phases are supported. Order is handled by `bootstrapper` and `BootOptions`.